docs/PUBLISH.md

```diff
@@ -1,11 +1,11 @@
+If you have any questions as you proceed through this guide, send Hank an email at henrymdoupe@gmail.com. 
 
-Once you've taken a look at the documentation, fill out the [publishing information template](TEMPLATE.md). The template will ask you to provide a title and overview for the new COMP app, code snippets for the three python functions, and information describing how to install this project and its resource requirements. Send the completed guide to Hank at henrymdoupe@gmail.com. He'll review the provided information to make sure that all of the criteria has been met. If all of the criteria has been met, it will be deployed to COMP. If there are any issues, then he will work with you to resolve them. Once the model has been deployed, you will have the opportunity to test it out. Since COMP is an open-source website, you can check Hank's work to verify that the code was plugged in correctly.
+The documentation is split into three parts. The [first part](IOSCHEMA.md) documents the inputs and outputs JSON schemas that your model will need to adopt for COMP to be able to generate input forms representing your model's default specification, validate user adjustments, and display model outputs. The [second part](ENDPOINTS.md) documents the python functions that will be used by COMP to get data from and submit data to your model. The third part is a [publishing information template](TEMPLATE.md) that asks you to provide a title and overview for your new COMP app, code snippets for the three python functions, and information describing your app's resource requirements and installation directions.
 
-If you have any questions about this guide, feel free to email Hank at henrymdoupe@gmail.com.
+Once you've completed the guide, send Hank a link to the completed template -- ideally committed to your project's repository if it is public -- at henrymdoupe@gmail.com. He'll review the  information and provide guidance if any of the criteria have not been met. Then your model will be deployed to COMP. Once the model has been deployed, you will have the opportunity to test it out. 
 
-For those who are interested in a more detailed explanation of how a model is published on COMP, feel free to checkout the [Technical Publishing Guide](TECHNICALPUBLISHING.md).
\ No newline at end of file
+For those who are interested in a more detailed explanation of the additional steps we  take to publish your model on COMP, feel free to checkout the [Technical Publishing Guide](TECHNICALPUBLISHING.md), and, since COMP is an open-source website, you can even follow along.
```

Comment:
Agreed, it seems more appropriate that the "open-source website" comment should go here.